spec/ics-1-ics-standard/README.md

```diff
@@ -0,0 +1,152 @@
+that individual blockchains in the Cosmos ecosystem may utilize their governance processes
+to approve or reject inter-chain standards).
+
+## Components
+
+An ICS consists of a header, synopsis, specification, history log, and copyright notice. All top-level sections are required.
+References should be included inline as links, or tabulated at the bottom of the section if necessary.
+
+### Header
+
+An ICS header contains metadata relevant to the ICS.
+
+#### Required fields
+
+`ics: #` - ICS number (assigned sequentially)
+
+`title` - ICS title (keep it short & sweet)
+
+`stage` - Current ICS stage, one of the following:
+- `proposal` - A standard in the "proposal" stage
+- `draft` - A standard in the "draft" stage
+- `candidate` - A standard in the "candidate" stage
+- `finalized` - A standard in the "finalized" stage
+
+See [README.md](../../README.md) for a description of the ICS acceptance stages.
+
+`category` - ICS category, one of the following:
+- `meta` - A standard about the ICS process
+- `ibc`  - A standard about the inter-blockchain communication system
+- `misc` - A standard about miscellaneous / auxiliary features, e.g. message signing
+
+`author` - ICS author(s) & contact information (in order of preference: email, GitHub handle, Twitter handle, other contact methods likely to elicit response).
+           The first author is the primary "owner" of the ICS and is responsible for advancing it through the standardization process.
+           Subsequent author ordering should be in order of contribution amount.
+
+`created` - Date ICS was first created (`YYYY-MM-DD`)
+
+`modified` - Date ICS was last modified (`YYYY-MM-DD`)
+
+#### Optional fields
+
+`requires` - Other ICS standards, referenced by number, which are required or depended upon by this standard.
+
+`required-by` - Other ICS standards, referenced by number, which require or depend upon this standard.
+
+`replaces` - Another ICS standard replaced or supplanted by this standard, if applicable.
+
+`replaced-by` - Another ICS standard which replaces or supplants this standard, if applicable.
+
+### Synopsis
+
+Following the header, an ICS should include a brief (~200 word) synopsis providing a high-level
+description of and rationale for the specification.
+
+### Specification
+
+The specification section is the main component of an ICS, and should contain protocol documentation, design rationale,
+required references, and technical details where appropriate.
+
+#### Sub-components
+
+The specification may have any or all of the following sub-components, as appropriate to the particular ICS. Included sub-components should be listed in the order specified here.
+
+- *Motivation* - A rationale for the existence of the proposed feature, or the proposed changes to an existing feature.
+- *Desired Properties* - A list of the desired properties or characteristics of the protocol or feature specified.
```

Comment:
Can we add "and expected effects when the properties are violated"? Implementors can expect what could go wrong when they don't follow the requirements(and write a failsafe code for it).

Reply:
Added!